src/js/components/__tests__/ConfigurationMapToggleValue-test.js

```diff
@@ -0,0 +1,28 @@
+const React = require("react");
+const ReactDOM = require("react-dom");
+const TestUtils = require("react-addons-test-utils");
+
+const ConfigurationMapToggleValue = require("../ConfigurationMapToggleValue");
+
+describe("ConfigurationMapToggleValue", function() {
+  it("should toggle the text value when clicked", function() {
+    const container = global.document.createElement("div");
+    const instance = ReactDOM.render(
+      <ConfigurationMapToggleValue
+        primaryValue="primary"
+        secondaryValue="secondary"
+      />,
+      container
+    );
+    const content = TestUtils.findRenderedDOMComponentWithClass(
+      instance,
+      "configuration-map-toggle-value"
+    );
+
+    expect(content.innerHTML).toContain("primary");
```

Comment:
This could be it's own test case. Like `it shows initially the primary value`. And then add a second case for the secondary value. It is usually better to only have a single expect per test case. So if the expect fails the developer directly knows what failed without looking into the test.

Reply:
Good suggestion, thanks, will change.